### PR TITLE
default link style

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -476,28 +476,44 @@ a:active {
     text-decoration: none;
 }
 
-p a {
+a {
     position: relative;
 }
 
-p a:after {
+a:after {
```
Comment (vinyll) [inline static/css/styles.css]: Are you sure you want this on all existing list of the whole site (including header, nav, footer, …)?

File: static/css/styles.css [hunk 2]
```diff
@@ -476,28 +476,44 @@ a:active {
...
+    left: 0;
+    bottom: 0;
+    width: 0;
+    transition: all .3s;
+}
+a:hover:after {
+    width: 100%;
+}
+
+.article-page article p a:after {
```
Comment (vinyll) [inline static/css/styles.css:498]: Why do you redo what you already have above?

File: static/css/styles.css (hunk 1 above)
Comment (johangiraud) [reply static/css/styles.css]: That's the default behavior, then I have to overwrite it if I want something for a specific element

File: static/css/styles.css (hunk 2 above)
Comment (johangiraud) [reply static/css/styles.css:498]: I don't understand what you're saying

Comment (vinyll) [reply static/css/styles.css:498]: you're doing here `content: ''` (line 498) that is already default in `a` (line 484).
Same for `position:absolute`, `bottom: 0`, …

Comment (johangiraud) [reply static/css/styles.css:498]: oh yes right

**Review by vinyll — APPROVED** [review]

**Pull request merged**